client/src/components/main/thread/AddThread.js

```diff
@@ -2,19 +2,63 @@ import React, { useState } from 'react';
 import styled from 'styled-components';
 import { Textarea } from './thread.style';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faFileExport } from '@fortawesome/free-solid-svg-icons';
+import { faReply } from '@fortawesome/free-solid-svg-icons';
 import { Tooltip } from '../../../theme/theme';
+import axios from 'axios';
+
+function AddThread({ dispatch, commentid }) {
+  const [reply, setReply] = useState('Reply...');
```

Comment:
Can  we have initial state be empty `""` and put a placeholder attribute on the `Textarea`? 'placeholder = "Reply..."`

Reply:
Yes, I can do that.